Drivers/Dxl/dxl.c

```diff
@@ -188,7 +188,18 @@ void rx_dxl_cb(module_t *module, msg_t *msg)
         {
             last = 0;
         }
-        dxl[last].val = position[find_id(module)];
+        angular_position_t value;
+        // convert data into deg
+        int i = find_id(module);
+        if (dxl_model[i] == AX12 || dxl_model[i] == AX18 || dxl_model[i] == XL320)
+        {
+            value = ((300.0 * (float)position[i]) / (1024.0 - 1.0)) - (300.0 / 2);
```

Comment:
```suggestion
            value = angular_position_from_deg(((300.0 * (float)position[i]) / (1024.0 - 1.0)) - (300.0 / 2));
```

Reply:
Using the "angular_position_from_deg" doesn't change anything because `angular_position_t` are actually stored into degree, but you shouldn't have to know it. A good practice to use OD structures such as `angular_position_t` is to always use a function to set and get it. Just to be sure to keep the good unit.